src/ProjectOrigin.Electricity.Client/ElectricityCommandBuilder.cs

```diff
@@ -40,7 +40,7 @@ public Task<CommandId> Execute(RegisterClient client)
         FederatedCertifcateId id,
         DateInterval inteval,
         string gridArea,
-        ShieldedValue gsrn,
+        ulong gsrn,
```

Comment:
The shielded value was dropped hence it is static throughout the lifecycle and hence also static as a shielded value. Shielded values are only worthwhile if they change throughout the certificate lifecycle as I understand it. It is still a sensitive piece of information and should be regarded as such.